thunder/tests/test_inplace_functionalization.py

```diff
@@ -455,3 +455,30 @@ def f(a, b):
     b_out = jitted(a, b)
     torch.testing.assert_close(b_out, b__out)
     assert b.data_ptr() == b_out.data_ptr()
+
+
+@instantiate(
+    dtypes=NOTHING,
+)
+def test_no_self_repeat_in_subsymbols(executor, device, _):
+
+    def f(a, b, c):
+        return a.add_(b, alpha=c)
```

Comment:
Will it work as expected if the `a.add_(b, alpha=c)` line is duplicated, I wonder?

Reply:
500b6da